src/sick_tim_common_usb.cpp

```diff
@@ -264,6 +264,7 @@ int SickTimCommonUsb::sendSOPASCommand(const char* request, std::vector<unsigned
 {
   if (device_handle_ == NULL) {
     ROS_ERROR("LIBUSB - device not open");
+    sleep(1);
```

Comment:
@jspricke The USB connection doesn't have a timeout so it occupies a core as fast as it can trying to reconnect - not very advantageous for a slow booting sensor or a sensor that's having a power/data cable issue.